tasks.py

```diff
@@ -53,3 +75,55 @@ def integrations_changed(ctx):
         if line:
             checks.add(line.split('/')[0])
     return checks
+
+
+@task(help={
+    'package': 'The package to upgrade throughout the integrations',
+    'version': 'The version of the package to pin',
+    'verbose': 'Whether or not to produce output',
+})
+def upgrade(ctx, package, version, verbose=False):
```

Comment:
can you put a docstring here? Without that you just get a blank output when you ask invoke for the commands
```
~/d/integrations-core  ofek/add-upgrade-task ◼ ❯❯❯ invoke -l                                                                                                                       ⏎
Available tasks:

  test      Run the tests for Agent-based checks
  upgrade
```